### PR TITLE
sidebars show officer and deputies

### PR DESCRIPTION
**Pull request by anna-wilson**
There is a bug in the code that shows officer info in the sidebars; it was only showing one officer, and sometimes that could be a deputy instead of the actual sitting officer. This pull request changes it so that it shows the officer and deputies, in the same way that the contacts page does.

### CODE REVIEW

File: _includes/officer-for-sidebar.html [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
 
-{% if role.report %}<p><a href="{{ role.report }}">How to report</a></p>{% endif %}
+  {% for officer in officers %}
 
-{% if role.desc %}
-<p>
-{{ role.desc }}
-</p>
-{% endif %}
+    {% unless officer.responsibility == " " %}
```
Comment (olleolleolle) [inline _includes/officer-for-sidebar.html:11]: Q: this has a space, not the empty string, is that intentional?

Comment (anna-wilson) [reply _includes/officer-for-sidebar.html:11]: It is; the regnum seems to insert a space for a primary officer.

Comment (olleolleolle) [reply _includes/officer-for-sidebar.html:11]: Came to think of 2 things:

- Would liquid have a `strip` function? A: yes!  https://shopify.github.io/liquid/filters/strip/
- Would liquid have an `empty?/blank?` function?

Comment (olleolleolle) [reply _includes/officer-for-sidebar.html:11]: In addition, I learned about the `== empty` check! https://shopify.github.io/liquid/basics/types/#emptydrop

**Pull request merged**